src/middleware/node/index.ts

```diff
@@ -50,7 +56,7 @@ export function createNodeMiddleware(
     } else if (typeof next === "function") {
       next();
     } else {
-      onUnhandledRequest(request, response);
+      onUnhandledRequest!(request, response);
```

Comment:
Because of L41, onUnhandledRequest should always be defined at this point, right?
```suggestion
      onUnhandledRequest(request, response);
```